packages/engine/Source/Core/createWorldTerrainAsync.js

```diff
@@ -24,11 +23,10 @@ import defaultValue from "./defaultValue.js";
  * }
  *
  * @example
- * // Create Cesium World Terrain with water and normals.
```

Comment:
Sorry for the confusion. Cesium World Terrain _does_ have a water mask. Only CWB doesn't.

Reply:
yuuup, PEBCAK, I _understood_ that but made the change too hastily :laughing: fixed now